frontend/packages/multicluster-sdk/src/api/search/types.ts

```diff
@@ -8,5 +8,5 @@ export type SearchResult<R extends K8sResourceCommon | K8sResourceCommon[]> = R
 
 export type UseMulticlusterSearchWatch = <T extends K8sResourceCommon | K8sResourceCommon[]>(
   watchOptions: WatchK8sResource,
-  advancedSearch?: { [key: string]: string }
+  advancedSearch?: { key: string; value: string }[]
```

Comment:
@Randy424 Maybe use `property` instead of `key` to better match search API terminology. And as discussed, allow `value` to optionally be an array.